diff --unified --recursive '--exclude=.pylint-disable.patch' original/agid.py patched/agid.py
--- original/agid.py
+++ patched/agid.py
@@ -216,7 +216,7 @@
     _handlers[handler_name] = Handler(handler_name, setup_fn, handle_fn)
 
 
-def sighup_handle(signum, frame):
+def sighup_handle(signum, frame): # pylint: disable=unused-argument
     log.debug("reloading core engine")
     _server.setup()
 
